tornado/ioloop.py

```diff
@@ -863,11 +868,14 @@ class PeriodicCallback(object):
 
     .. versionchanged:: 5.1
        The ``jitter`` argument is added.
+
+    .. versionchanged:: 6.2
+       The ``callback`` argument may be a coroutine.
```

Comment:
The change is more subtle than that: This PR adds support for *native* coroutines. Decorated coroutines have always been supported by the code (I think; haven't tested it. The mypy annotations would reject them but I'm pretty sure they'd still work). However, this changes their semantics: they were previously fire-and-forget (so a slow coroutine could end up with multiple copies of the task running concurrently) while now they affect scheduling (so a slow coroutine will reduce the number of calls made per unit of time). 

This is a potentially breaking change so we don't want to make it by accident. I think it's probably the right move but it needs to be a deliberate decision and documented accordingly.

Reply:
Native coroutines worked too, I used them for a long time, but they were fire-and-forget as you say. I'll update this to be more specific.